ui/src/main/java/pl/senla/hotel/Main.java

```diff
@@ -1,21 +1,34 @@
 package pl.senla.hotel;
 
 import lombok.extern.slf4j.Slf4j;
-import pl.senla.hotel.application.di.DIApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import pl.senla.hotel.ui.main.StartMenuMainDB;
 
 import java.lang.reflect.InvocationTargetException;
 
 import static pl.senla.hotel.constant.ApplicationContextConstant.START_APPLICATION;
 
 @Slf4j
+@SpringBootApplication
 public class Main {
 
     public static void main(String[] args)
             throws IllegalAccessException, InvocationTargetException, NoSuchMethodException,
             InstantiationException, InterruptedException {
 
-        log.debug(START_APPLICATION);
-        DIApplication.getApplication().run();
+        log.info(START_APPLICATION);
+
+        // previous version 1
+//        DIApplication.getApplication().run();
+
+//        version 3:
+        ApplicationContext ctx = new AnnotationConfigApplicationContext(SpringConfig.class);
+        StartMenuMainDB startMenuMainDB = ctx.getBean(StartMenuMainDB.class);
```

Comment:
please read about post construct annotation and move it into StartMenuMainDB class. It's a very bad practice to use app context in your business logic there is always a way to solve it in a 'right' way

Reply:
ready